pkg/osbuild/dnf_module_config_stage.go

```diff
@@ -0,0 +1,62 @@
+package osbuild
+
+import (
+	"github.com/osbuild/images/pkg/rpmmd"
+)
+
+type DNFModuleConfig struct {
+	Name     string   `json:"name,omitempty"`
+	Stream   string   `json:"stream,omitempty"`
+	State    string   `json:"state,omitempty"`
+	Profiles []string `json:"profiles"`
+}
```

Comment:
_This is not a comment for this PR directly_

The lack of `omitempty` on `Profiles` lead me to look at the stage schema and I noticed that there are no fields marked `required` there.  The stage itself however accesses properties from the stage options dictionary without using `.get()` but directly with `[]`.
We should fix the stage to use `.get()` for the non-required options, or make them required.  The latter is a backwards-incompatible change, but given the recency of the stage itself, I think that might be okay in this case.